source/localizable/models/relationships.md

```diff
@@ -43,7 +43,7 @@ import Model from 'ember-data/model';
 import { belongsTo } from 'ember-data/relationships';
 
 export default Model.extend({
-  post: belongsTo('post')
+  blogPost: belongsTo('blog-post')
```

Comment:
Having `post` and `blog-post` differ from example to example is needlessly confusing, and will trip people up, so it needs to be consistent.
The Guides do have an implied narrative, we try to lessen the cognitive overload by trying to make sure stuff is in the same "universe".